cli/generate.go

```diff
@@ -24,11 +27,30 @@ func Generate() cli.Command {
 	}
 }
 
+const (
+	jsonFormat = "json"
+	yamlFormat = "yaml"
+)
+
+type configFormatter func(*shrub.Configuration) ([]byte, error)
+
+func generatedConfigFormats() map[string]configFormatter {
```

Comment:
should this be `generateConfigFormatters()` instead?

Reply:
I changed this to just take the format as a parameters and switch-case on it.